dvc/ignore.py

```diff
@@ -27,12 +20,13 @@ def __call__(self, root, dirs, files):
         raise NotImplementedError
 
 
-class DvcIgnoreFromFile(DvcIgnore):
-    def __init__(self, ignore_file_path, ignore_handler):
+class DvcIgnorePatterns(DvcIgnore):
+    def __init__(self, ignore_file_path):
         self.ignore_file_path = ignore_file_path
```

Comment:
Judging by `__eq__`, we expect only abs path here, right? Should we put a check/assert here? Otherwise it might get nasty with `__eq__` saying True for two relative paths.